File: src/main/java/no/nav/fo/veilarbregistrering/domain/Profilering.java
package no.nav.fo.veilarbregistrering.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import lombok.experimental.Accessors;

import java.util.Objects;

@Data
@Accessors(chain = true)
@ToString
@EqualsAndHashCode
public class Profilering {
    boolean jobbetSammenhengendeSeksAvTolvSisteManeder;
    int alder;
    Innsatsgruppe innsatsgruppe;
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/profilering/Profilering.kt
package no.nav.fo.veilarbregistrering.profilering

data class Profilering(
    var innsatsgruppe: Innsatsgruppe,
    var alder: Int,
    var jobbetSammenhengendeSeksAvTolvSisteManeder: Boolean,
) {

    override fun toString(): String {
        return "Profilering(jobbetSammenhengendeSeksAvTolvSisteManeder=$jobbetSammenhengendeSeksAvTolvSisteManeder, alder=$alder, innsatsgruppe=$innsatsgruppe)"
    }
}